pages/price-feeds/best-practices.mdx

```diff
@@ -70,6 +73,18 @@ This analogy suggests two simple solutions to races:
    The protocol can require a short delay between the two transactions, and the user's request gets fulfilled at the Pyth price as of the second transaction.
    This technique gives the protocol extra time to observe price changes, giving it a head start in the latency race.
 
+### Latency Mitigations for Derivative Protocols
+
+Derivative protocols are encouraged to consider the following strategies to mitigate the impact of oracle latency:
+
+1. **Use a Delayed Settlement**: Derivative protocols can introduce a delay between the time a ordere is created and the time it is executed. This delay gives the protocol time to observe price changes and reject trades/transactions that profit over latency.
+   Moreover, As mentioned above the protocol can introduce a short delay (~5 seconds) between the time a user submits the tx to the application and the keeper submits the tx to the blockchain.
+   The keeper can use Pyth Benchmark to get the price of `t - 5` seconds, where `t` is the current time and `t - 5` is the time when the user submitted the tx to the dApp.
+   Now the keeper can use [`parsePriceFeedUpdates()`](https://api-reference.pyth.network/price-feeds/evm/parsePriceFeedUpdates) to parse the prices and submit to prevent price frontrunning.
```

Comment:
This description of delayed settlement is confusing. The core idea of delayed settlement is that when a user places an order at time `t`, the trade executes using the pyth price at time `t`. (Now, this price is only available from pyth after a short delay, hence the name.) We should explain delayed settlement in these terms though because it's much clearer why this works.

Reply:
Rewrote it. Still feel it could be better. 🤔